oap-server/server-bootstrap/src/main/resources/alarm-settings.yml

```diff
@@ -81,3 +81,16 @@ dingtalkHooks:
   webhooks:
 #    - url: https://oapi.dingtalk.com/robot/send?access_token=dummy_token
 #      secret: dummysecret
+
+feishuHooks:
+  textTemplate: |-
+    {
+      "msg_type": "text",
+      "content": {
+        "text": "Apache SkyWalking Alarm: \n %s."
+      },
```

Comment:
This may cause the OAP startup to fail due to a YAML parse failure. You need to remove this if it is unnecessary params, as follows:
```yaml
feishuHooks:
  textTemplate: |-
    {
      "msg_type": "text",
      "content": {
        "text": "Apache SkyWalking Alarm: \n %s."
      }
    }
```

Reply:
and others work well.

![image](https://user-images.githubusercontent.com/12468337/104831221-d0cd9b00-58c1-11eb-8341-6ed7dde5cd84.png)